src/app/config/config.py

```diff
@@ -1,28 +1,62 @@
 from config.config_helpers import get_value_from_env
 from pydantic import BaseSettings
 
+TIME_FORMAT_STRING = "%Y-%m-%dT%H:%M:%SZ"
+
 
 class Config(BaseSettings):
     def __init__(self):
         super().__init__()
         self.CONF = get_value_from_env("CONF")
-        self.TIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+        self.TIME_FORMAT = TIME_FORMAT_STRING
         self.DATASET_BUCKET_NAME = get_value_from_env("DATASET_BUCKET_NAME")
 
     CONF: str
-    TIME_FORMAT: str = "%Y-%m-%dT%H:%M:%SZ"
+    TIME_FORMAT: str = TIME_FORMAT_STRING
     DATASET_BUCKET_NAME: str
 
 
+class IntegrationTests(BaseSettings):
+    def __init__(self):
+        super().__init__()
+        self.CONF = get_value_from_env("CONF")
+        self.TIME_FORMAT = TIME_FORMAT_STRING
+        self.API_URL = get_value_from_env("API_URL", "localhost")
+        self.DATASET_BUCKET_NAME = get_value_from_env(
+            "DATASET_BUCKET_NAME", "testDatasetBucket"
+        )
+        self.SCHEMA_BUCKET_NAME = get_value_from_env(
+            "SCHEMA_BUCKET_NAME", "testSchemaBucket"
+        )
```

Comment:
Very minor but you're using camel case for these test vars but should really be snake case for Python test_schema_bucket